irohad/execution/impl/command_executor.cpp

```diff
@@ -63,90 +63,11 @@ namespace iroha {
-          command_name);
-    }
-    auto account_asset =
-        queries->getAccountAsset(creator_account_id, command.assetId());
-
-    auto new_balance = command_amount | [this](const auto &amount) {
-      return amount_builder_.precision(amount->precision())
-          .intValue(amount->intValue())
-          .build();
-    };
-    using AccountAssetResult =
-        expected::Result<std::shared_ptr<shared_model::interface::AccountAsset>,
-                         iroha::CommandError>;
-    auto account_asset_new = new_balance.match(
-        [this, &account_asset, &command_name, &command](
-            const expected::Value<
-                std::shared_ptr<shared_model::interface::Amount>>
-                &new_balance_val) -> AccountAssetResult {
-          expected::PolymorphicResult<shared_model::interface::AccountAsset,
-                                      std::string>
-              result;
-          if (account_asset) {
-            result = (*new_balance_val.value + account_asset.value()->balance())
-                | [this, &command](const auto &balance) {
-                    return account_asset_builder_.balance(*balance)
-                        .accountId(creator_account_id)
-                        .assetId(command.assetId())
-                        .build();
-                  };
-          } else {
-            result = account_asset_builder_.balance(*new_balance_val.value)
-                         .accountId(creator_account_id)
-                         .assetId(command.assetId())
-                         .build();
-          }
-          return result.match(
-              [](expected::Value<
-                  std::shared_ptr<shared_model::interface::AccountAsset>>
-                     &new_account_asset_val) -> AccountAssetResult {
-                return expected::makeValue(new_account_asset_val.value);
-              },
-              [&command_name](const auto &error) -> AccountAssetResult {
-                return makeCommandError(*error.error, command_name);
-              });
-        },
-        [&command_name](const auto &error) -> AccountAssetResult {
-          return makeCommandError(
-              "amount builder failed. reason " + *error.error, command_name);
-        });
-
-    return account_asset_new.match(
-        [&](const expected::Value<
-            std::shared_ptr<shared_model::interface::AccountAsset>>
-                &account_asset_new_val) -> CommandResult {
-          return makeCommandResult(
-              commands->upsertAccountAsset(*account_asset_new_val.value),
-              command_name);
-        },
-        [&command_name](const auto &account_asset_error) -> CommandResult {
-          return makeCommandError("account asset builder failed. reason "
-                                      + account_asset_error.error.toString(),
-                                  command_name);
-        });
+    auto result = commands->addAssetQuantity(creator_account_id,
```

Comment:
This class seems redundant now. Maybe create Postgres command executor and remove wsv_command altogether?

Reply:
We can make postgres_command executor and keep not-rewritten functions along with changed ones together until migration is complete